.env.example

```diff
@@ -0,0 +1,27 @@
+ENVIRONMENT=todo
+
+TEST_REPO_NAME=todo
+
+INSTALLATION_LOGIN=todo
+
+MAIN_TICK_SECS=todo
+
+PRIVATE_KEY_PATH=todo
+
+DB_PATH=todo
+
+WEBHOOK_SECRET=todo
+WEBHOOK_PORT=todo
+
+BAMBOO_TOKEN=todo
```

Comment:
I know that you're going to be removing some of these in your refactor branch but the general idea on having this example file is fine to keep IMO

Reply:
Can also probably move alot of these into `constants.rs` (which you've probably already done)